deploy/clowd-environment.yml

```diff
@@ -0,0 +1,36 @@
+--- 
```

Comment:
I'm not sure you need the clowd-environment here at all

Reply:
OK I will remove it